tests-infra/Cargo.toml

```diff
@@ -0,0 +1,14 @@
+[package]
```

Comment:
What's the purpose of this new crate?  It appears to duplicate a lot of existing code.

Reply:
i wanted to avoid code duplication and share code across `tests` and `tests-alloc`

Reply:
it's possible that that has all gotten messed up by the merge, i will take a look. i think the best course of action is likely to nuke tests-alloc completely